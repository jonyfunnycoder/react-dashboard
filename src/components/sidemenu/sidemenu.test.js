import React from 'react';
import renderer from "react-test-renderer";
import SideMenu from "./sidemenu";

test('renders sidemenu component correctly ', () => {
    const tree = renderer
        .create(<SideMenu />)
        .toJSON();

    expect(tree).toMatchSnapshot();
});
