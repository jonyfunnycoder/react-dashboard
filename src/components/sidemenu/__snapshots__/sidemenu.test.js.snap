// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders sidemenu component correctly  1`] = `
<div
  className="no-gutter fixed side-menu col-md-1"
>
  <div
    className="side-menu-user flex flex-full-center"
  >
    <span>
      S
    </span>
  </div>
</div>
`;
